src/components/EventDateFilter/EventDateFilter.js

```diff
@@ -1,16 +1,45 @@
 import React, { PropTypes } from 'react';
 import DatePicker from 'react-datepicker';
+import moment from 'moment';
 
 import styles from './EventDateFilter.sass';
 
+const dateFormatToday = 'MMM D';
+const dateFormatMain = 'ddd MMM D';
+
+
+function isStartDateToday(startDate) {
+  const dateToday = moment();
+  return moment(startDate).isSame(dateToday, 'day');
+}
+
+function generateDateFormat(startDate) {
+  if (isStartDateToday(startDate)) {
+    return dateFormatToday;
+  }
+
+  return dateFormatMain;
```

Comment:
DERR, yeah, that was left over from an earlier approach at this, so the function declarations I had at the top of the file are no longer necessary. 

The reason for changing the format was that the mockup called for today's date to display as the word, "TODAY", so I have that set as the default display date for whenever the page loads. Otherwise, today's date would show up as "SUN APR 16", instead of "TODAY APR 16". It wasn't totally clear what the limitations of the formatting options are with react-datepicker, but I couldn't find any sort of format I could use in the `dateFormat` prop that would have automatically use the string, "TODAY" on page load and also when the user selects the current day on the date picker. 

So my solution was to use the today-specific format using the default prop of ``TODAY ${moment().format('MMM D')}`` , and then when the user actually selects the current day, to make that value null, so we get my default property value again. 

Some of this might be reworked now that the API is getting changed, but as far as simply displaying the date in the filter section, I think this should work. Thoughts?